internal/providers/terraform/aws/ecs_task_definition.go

```diff
@@ -0,0 +1,24 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// This is a free resource but needs it's own custom registry item to specify the custom ID lookup function.
+func getECSTaskDefinitionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:    "aws_ecs_task_definition",
+		NoPrice: true,
+		Notes:   []string{"Free resource 2."},
```

Comment:
```suggestion
		Notes:   []string{"Free resource."},
```

Reply:
> but it doesn't work if the user also specifies a revision, i.e:

I guess the issue is we don't have access to revision when parsing HCL because it's generated by aws.  The arn includes family + revision, so maybe the best thing we can do is add a note that to use an arn ref to get an exact version.  Or maybe even a warning if we detect the link includes the ":<revision>" part?

Reply:
> Suggested change 
Thanks!

> I guess the issue is we don't have access to revision...

Yeah I was thinking if they pass `family:revision` we should just use the task definition in the code that has that family, since most likely they are doing something like `family = "${aws_ecs_task_definition.task1.family}:${aws_ecs_task_definition.task1.revision}`. The problem is, as far as I remember, the way we match references means we match on the whole value of the attribute, so we can't match on the part before the `:`.